docs/changelog/85326.yaml

```
@@ -0,0 +1,6 @@
+pr: 85326
+summary: "Fix: fail node start if LDAP/AD realm has bind DN set but is missing bind password"
```

The `Fix:` prefix is unnecessary.
The PR is labelled as `>bug`, so the changelog file already knows that this is `type: bug` and the release notes will include it in the relevant section.
```suggestion
summary: "Fail node start if LDAP/AD realm has bind DN set but is missing bind password"
```